lib/AccessoryManager.js

```diff
@@ -500,6 +507,200 @@ class AccessoryManager {
     )
   }
 
+  /**
+   * Configure the "Light" service for the accessory
+   */
+  _configureDimLightService () {
+    const lightBulbService = this._getService(this._hapService.Lightbulb)
+    const onCharacteristic = lightBulbService.getCharacteristic(this._hapCharacteristic.On)
+    const brightnessCharacteristic = lightBulbService.getCharacteristic(this._hapCharacteristic.Brightness)
+
+    // Setup handler for when the value for the "On" characteristic is requested / set by HomeKit
+    const onCharacteristicName = 'On'
+    const onCharacteristicZwaveNodeValueCriteria = {
+      commandClass: COMMAND_CLASS_SWITCH_MULTILEVEL,
+      valueIndex: SWITCH_BINARY_INDEX_SWITCH
+    }
+
+     const onCharacteristicValueSet = value => {
+       var setValue = brightnessCharacteristic.value
+       if (value) {
+         if (setValue === 1){
+           setValue = 99
+         }
+         else if (setValue === 100) {
+           setValue = 99
+         }
+       } else {
+         setValue = 0
+       }
+       return setValue
```

Comment:
Shouldn't this be returning a `boolean` value rather than a `number`? (On value is normally `true` or `false`)